src/main.rs

```diff
@@ -203,28 +229,26 @@ fn do_ceremony(
     print_dev: &Path,
     args: &Args,
 ) -> Result<()> {
+    // this is mut so we can zeroize when we're done
```

Comment:
Alternatively, using Zeroizing::new() so it zeroizes on drop.

Reply:
Solid idea. I should do this in a few places. Will do all that in a separate PR.